orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/MonitoredDeployBaseTask.java

```diff
@@ -267,22 +263,21 @@ private MonitoredDeployStageData getStageContext(StageExecution stage) {
     return stage.mapTo(MonitoredDeployStageData.class);
   }
 
-  private String getRetrofitLogMessage(Response response) {
-    if (response == null) {
+  private String getRetrofitLogMessage(SpinnakerServerException spinnakerException) {
+    if (!(spinnakerException instanceof SpinnakerHttpException)) {
       return "<NO RESPONSE>";
     }
 
+    SpinnakerHttpException httpException = (SpinnakerHttpException) spinnakerException;
     String body = "";
     String status = "";
     String headers = "";
+    ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Let's only allocate one of these instead of making a new one every time this method is called.

Reply:
- Changed the variable initialisation of ObjectMapper to instance variable of the class.